nsupdate/__init__.py

```diff
@@ -62,4 +62,4 @@ def __str__(self):
         return version_str
 
 
-version = Version(0, 12, 0)
+version = Version(0, 13, 0)
```

Comment:
we modified the milestone for the respective "add django 1.11 support" to 0.12, so guess this change above is not needed.